srv/components/misc_pkgs/kafka/sanity_check.sls

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+{% for i in range(5) -%}
+Test Kafka-zookeeper:
+    cmd.run:
+        -name: test 1 -le $(ps ax | grep java | grep -i QuorumPeerMain | grep -v grep | awk '{print $1}' | wc -l)
+        - test.sleep:
```

Comment:
Can we have something like - 
https://github.com/Seagate/cortx-prvsnr/blob/a8a677d63646e4c626897f40ecbabb1106526317/srv/components/system/storage/multipath/config.sls#L92
```
Test kafka:
  cmd.run:
    - name: test 1 -le $(ps ax | grep ' kafka\.Kafka ' | grep java | grep -v grep | awk '{print $1}' | wc -l)
    - retry:
        attempts: 5
        until: True
        interval: 10
```